assets/docs/CallmeMehdi/models/AraBERT/1.md

```diff
@@ -0,0 +1,23 @@
+# Module CallmeMehdi/arabert/1
+
+AraBERT is an Arabic pretrained lanaguage model based on Google's BERT architechture.
+
+<!-- asset-path: https://firebasestorage.googleapis.com/v0/b/gsoc-tf.appspot.com/o/arabert.tar.gz?alt=media&token=095808fe-9364-4330-938a-5961bc828191 -->
+<!-- task: text-embedding -->
+<!-- fine-tunable: true -->
+<!-- format: saved_model_2 -->
+
+## Overview
+
+These results are obtained from pre-training BERT specifically for the Arabic language in the pursuit of achieving the same success that BERT did for the English language.The performance of AraBERT is compared to multilingual BERT from Google and other state-of-the-art approaches
```

Comment:
Can you add more detail here? You mention some performance comparisons. It would be great to add some sections with details, like:

```markdown
## Performance

AraBERT was tested on ...

Results:

 Task | Accuracy | ...
------|----------|----
Task1 | 98.9%    |----
```